src/System.Net.Http/src/uap/System/Net/HttpClientHandler.cs

```diff
@@ -701,6 +657,15 @@ private void SetOperationStarted()
                     _rtFilter.IgnorableServerCertificateErrors.Add(RTChainValidationResult.Untrusted);
                     _rtFilter.IgnorableServerCertificateErrors.Add(RTChainValidationResult.WrongUsage);
                     _rtFilter.ServerCustomValidationRequested += RTServerCertificateCallback;
+
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.Expired);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.IncompleteChain);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.InvalidName);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.RevocationFailure);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.RevocationInformationMissing);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.Untrusted);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.WrongUsage);
+                    _rtFilterWithNoCredentials.ServerCustomValidationRequested += RTServerCertificateCallback;
```

Comment:
And this pattern is also the same for just the .NET layer itself.  We have a TLS/SSL certificate validator callback defined per handler.  So, multiple requests (at the same time perhaps) can happen against the single callback method.